### DISCUSSION TITLE
libreSDR port

### DISCUSSION THREAD

**Discussion by sdr-fun** [open-sdr/openwifi#400] [Ideas]
Please add port for LibreSDR hardware.  This is a cheap and high-quality platform, it successfully replaces the usual plutosdr and is available to many users. Information about it is https://github.com/day0wl/libresdr-fw 

I'm trying to make a port myself, but my experience is not enough to do it quickly and without difficulty. I corrected the hw project to match the board parameters, generated a bitstream, but now it’s difficult to create a correct devicetree, I’m faced with a misunderstanding where the SDR driver comes from, because when exporting from Vivado and Vitis there is nothing similar, but in the ready-made ports of the repository the SDR is present.

**Comment by pavelyazev** [comment]
Hi, I have implemented a port for LibreSDR hardware. You can try [it](https://github.com/pavelyazev/openwifi-libresdr)

**Reply by sdr-fun** [reply]
Hi! Thank you! True, I still managed to launch openwifi on the libreSDR platform, although I spent a lot of time, but I gained new knowledge)) Thank you for your work, Pavel!
